crates/aptos-api-tester/src/token_client.rs

```diff
@@ -0,0 +1,458 @@
+// Copyright © Aptos Foundation
```

Comment:
This code probably belongs somewhere else, like the SDK crate or the REST client crate (even if it is for token v1). What do you think @gregnazario? If we don't move it now we should at least make an issue and TODO in this file to remind us to do it later.

Reply:
I actually asked him to move it from the Rust SDK, as we need to rethink the rust SDK structure